notebooks-auth/test_cowbird_jupyter.ipynb

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
<p>Add a URL reference to the optional component (expected after merge).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>